Twig/Extension/FormatterMediaExtension.php

```diff
@@ -109,4 +109,14 @@ public function path($media, $format)
     {
         return $this->getTwigExtension()->path($media, $format);
     }
+
+    /**
+     * NEXT_MAJOR: Remove this method when bumping requirements to twig 2.
+     *
+     * {@inheritdoc}
+     */
+    public function getName()
```

Comment:
This will not be needed on twig 1.x when SonataFormatter bundle will be released.

Extending from \Twig_Extension instead of interface gives you this function already when you need it.